src/App.js

```diff
@@ -1,5 +1,15 @@
 import React from "react";
+import { Route, Routes } from "react-router-dom";
+import BookingHome from "./components/MovieBooking/BookingHome";
+import Moviehome from "./components/MovieList/Moviehome";
 
 export default function App() {
-  return <h1>Hello world!</h1>
-}
\ No newline at end of file
+  return (
+    <div>
+      <Routes>
+        <Route path="/" element={<Moviehome />} />
+        <Route path="/booking/:id/:path" element={<BookingHome />} />
```

Comment:
The redirected url when a movie is clicked - `http://localhost:3000/booking/616037/pIkRyD18kl4FhoCNQuWxWu5cBLM.jpg` The url should be `http://localhost:3000/booking/616037/thor-love-and-thunder`